docs/src/install/index.md

```diff
@@ -4,200 +4,213 @@ tags: [Installation]
 
-:::note
-If you want to get a specific version, append the `<version>` 3 digits (e.g. `0.7.3`):
+To install or update to the latest version, run the following command:
 
-```sh
-curl -fsSL https://raw.githubusercontent.com/thin-edge/thin-edge.io/main/get-thin-edge_io.sh | sudo sh -s <version>
+```sh tab={"label":"curl(sudo)"}
+curl -fsSL https://thin-edge.io/install.sh | sudo sh -s
 ```
-:::
 
-If you prefer to have a little more control over the installation or the script did not work for you,
-please go on with the [manual installation steps](#manual-installation-steps).
-
-## Upgrade via script
+```sh tab={"label":"curl(root)"}
+curl -fsSL https://thin-edge.io/install.sh | sh -s
+```
 
-If you already have `thin-edge.io` on your device, to upgrade `thin-edge.io`,
-the easiest way is to use the same script as the installation. Follow the steps below.
-There is no need to remove old version.
+```sh tab={"label":"wget(sudo)"}
+wget -O - https://thin-edge.io/install.sh | sudo sh -s
+```
 
-:::note
-For version **0.7.7 or lower, to upgrade `thin-edge.io` from these versions,
-all thin-edge.io components **must be stopped** before upgrading.
+```sh tab={"label":"wget(root)"}
+wget -O - https://thin-edge.io/install.sh | sh -s
+```
 
-The components are:
-`tedge-mapper-c8y`, `tedge-mapper-az`, `tedge-mapper-collectd`, `tedge-agent`, `tedge-watchdog`, `c8y-log-plugin`, `c8y-configuration-plugin`.
+**Updating in the future**
 
-To stop `tedge-mapper-c8y`, `tedge-agent`, `tedge-mapper-az`, `tedge-mapper-aws`, you can simply run the commands below.
+If you only want to use the install.sh script for the initial installation and not for future updates, then you can update thin-edge.io and its components using your distribution's package manager, as the install.sh script configures the package manager settings for you.
 
-```sh
-sudo tedge disconnect c8y
-sudo tedge disconnect az
-sudo tedge disconnect aws
+```sh tab={"label":"Debian/Ubuntu"}
+sudo apt-get update
+sudo apt-get install tedge-full
 ```
 
-To stop each component one by one, this is an example how to stop them with `systemctl`:
+```sh tab={"label":"RHEL/Fedora/RockyLinux"}
+sudo dnf install --best tedge-full --refresh
+```
 
-```sh
-sudo systemctl stop tedge-mapper-c8y
-sudo systemctl stop tedge-agent
-sudo systemctl stop c8y-log-plugin
+```sh tab={"label":"Alpine"}
+sudo apk update
+sudo apk add --no-cache tedge-full
 ```
-:::
 
-Run `get-thin-edge_io.sh` script as below to upgrade to the latest version.
+:::info
+The update only works if the tarball package manager was not used. The tarball installation does not use a package manager, as the install.sh script handles the downloading and extracting of the thin-edge.io tarball's contents. In this case, the easiest option is to continue using the install.sh script.
```

Comment:
One might add that the `install.sh` script works fine to update thin-edge when already installed.

By the way, what if thin-edge has been installed with a different script/method?

Reply:
> One might add that the install.sh script works fine to update thin-edge when already installed.

Though I would have assumed that wasn't necessary because the section header is called "Install/Update"...but if it is not clear, then I'm happy to add a sentence to that effect.

> By the way, what if thin-edge has been installed with a different script/method?

It might have some issues if you want from the tarball to auto detection, as the auto detection could find a package manager and install it via that. Though the tarball section does show an example how to explicitly install it via the tarball (rather than using auto detection).

Though technically, overwriting it should not be a problem, if it it was a problem, then the package manager would throw an error explaining why (at least for tarball -> package manager, not the other way around).